rfd/XXXX-tsh-upgrade.md

```diff
@@ -0,0 +1,117 @@
+---
+authors: Bernard Kim (bernard@goteleport.com)
+state: draft
+---
+# RFD XXXX - tsh upgrade
+
+## Required Approvers
+
+* Engineering: ...
+* Infra: ...
+* Kube: ...
+
+## What
+
+A tsh subcommand that will prompt the user to upgrade if the version of tsh is older than the cluster version.
```

Comment:
Should this RFD also cover `tctl`? From what I understand, Cloud users have to use a local `tctl` to perform administration operations.

If that's the case, will `tctl upgrade` work? Will `tsh upgrade` also update `tctl`?

Reply:
Yes, tctl should be in scope.